src/main/resources/youtube/settings/xml/revanced_prefs.xml

```diff
@@ -370,6 +370,7 @@
             <SwitchPreference android:title="@string/revanced_overlay_button_always_repeat_title" android:key="revanced_overlay_button_always_repeat" android:summary="@string/revanced_overlay_button_always_repeat_summary" />
             <SwitchPreference android:title="@string/revanced_overlay_button_copy_video_url_title" android:key="revanced_overlay_button_copy_video_url" android:summary="@string/revanced_overlay_button_copy_video_url_summary" />
             <SwitchPreference android:title="@string/revanced_overlay_button_copy_video_url_timestamp_title" android:key="revanced_overlay_button_copy_video_url_timestamp" android:summary="@string/revanced_overlay_button_copy_video_url_timestamp_summary" />
+            <SwitchPreference android:title="@string/revanced_overlay_button_mute_volume_title" android:key="revanced_overlay_button_mute_volume" android:defaultValue="false" android:summary="@string/revanced_overlay_button_mute_volume_summary" />
```

Comment:
The default value of the setting is already specified in integrations, so `android:defaultValue` is not needed in the xml